Monika After Story/game/script-topics.rpy

```diff
@@ -18041,3 +18041,76 @@ label monika_overton:
     m 1eua "I {i}could{/i} finish this story right up to Current Norm, but I just wanted to describe it at a basic level in order to convey an example of how it can work."
     m 1huu "Thanks for listening~"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_upset_apologize",
+            category=['you','monika'],
+            prompt="Something you did upset me.",
+            pool=True,
+            unlocked=True
+        )
+    )
+
+label monika_upset_apologize:
+    $ mas_loseAffection()
+    if mas_isMoniBroken():
+        m "...{w=1}Oh."
+        pause 1.0
+        m "Okay."
+        return
+    m 6rkc "...{w=1}Oh."
+    m 5lkc "Was it what I just said?"
+    menu:
+        "It was what you just said.":
+            m 1dkc "Okay."
+            if not mas_isMoniUpset(lower=True):
+                m 1dkd "Thank you for telling me..."  
+            m 2fkd "I promise I won't say something like that again."
+            m 2dkd "I'd never want to do something to hurt you, or make you uncomfortable, [player]."
+            m 3essdld "Also, please remember that you can press the 'X' key if you don't want me to bring something up again."
+
+        "It was a point you made.":
+            m 1dkc "Okay."
+            if not mas_isMoniUpset(lower=True):
+                m 1dssdld "Thank you for telling me..."
+            m 1rud "I wish I could hear your arguments right now."
+            m 7msd "But I guess all I can do is do a bit more research and thinking about what I've said recently."
+            m 3eka "Please remember that you can press the 'X' key if you don't want me to bring something up again!"
```

Comment:
I'd personally use something like 'you can always ask me not to bring up something' instead, I believe it would be less immersion-breaking.